client/src/app/site/pages/meetings/modules/projector/modules/slides/components/wifi-access-data-slide/components/wifi-access-data-slide/wifi-access-data-slide.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component } from '@angular/core';
+
+import { BaseSlideComponent } from '../../../../base/base-slide-component';
+import { WifiAccessDataSlideData } from '../../wifi-access-data-slide-data';
+
+@Component({
+    selector: `os-wifi-access-data-slide`,
+    templateUrl: `./wifi-access-data-slide.component.html`,
+    styleUrls: [`./wifi-access-data-slide.component.scss`]
+})
+export class WifiAccessDataSlideComponent extends BaseSlideComponent<WifiAccessDataSlideData> {
+    public get ssid(): string {
+        return this.data.data.users_pdf_wlan_ssid;
+    }
+
+    public get password(): string {
+        return this.data.data.users_pdf_wlan_password;
+    }
+
+    public get encryption(): string {
+        switch (this.data.data.users_pdf_wlan_encryption) {
+            case `WEP`:
+                return `WEP`;
+            case `WPA`:
+                return `WPA/WPA2`;
+            case `nopass`:
+                return `No encryption`;
+            default:
+                return undefined;
+        }
+    }
+
+    public get qrCodeText(): string {
+        return `WIFI:S:` + this.ssid + `;T:` + this.data.data.users_pdf_wlan_encryption + `;P:` + this.password + `;;`;
```

Comment:
Not entirely sure but I think `ssid` and `password` should be escaped.

Reply:
For reference: https://github.com/zxing/zxing/wiki/Barcode-Contents#wi-fi-network-config-android-ios-11